### ISSUE TITLE
Accessibility Issue : Here UIAccessibilityTraits.button.  should be  UIAccessibilityTraits.Link

### ISSUE THREAD

**Issue by akshayjain14** [psharanda/Atributika#171]
private func addDetectionAreaButton(frame: CGRect, detection: Detection, text: String) {
        let button = DetectionAreaButton(detection: detection)
        button.accessibilityLabel = text
        button.isAccessibilityElement = true
        #if swift(>=4.2)
        button.accessibilityTraits = UIAccessibilityTraits.button
        #else
        button.accessibilityTraits = UIAccessibilityTraitButton
        #endif
        
        button.isUserInteractionEnabled = state.isEnabled
        button.addTarget(self, action: #selector(handleDetectionAreaButtonClick), for: .touchUpInside)
        detectionAreaButtons.append(button)
        
        button.onHighlightChanged = { [weak self] in
            self?.state.detection = $0.isHighlighted ? $0.detection : nil
        }

**Comment by psharanda** [comment]
@akshayjain14 see my comment in https://github.com/psharanda/Atributika/pull/172
